Frameworks/UIKit/UITextField.mm

```diff
@@ -352,19 +348,32 @@ - (void)setBackgroundColor:(UIColor*)color {
  @Status Interoperable
 */
 - (UIColor*)backgroundColor {
-    return _backgroundColor;
+    WUXMSolidColorBrush* colorBrush = nil;
+    if (_secureTextMode) {
+        colorBrush = rt_dynamic_cast<WUXMSolidColorBrush>(_passwordBox.background);
+        if (colorBrush != nil) {
+            return ConvertWUColorToUIColor(colorBrush.color);
+        } else {
+            return nil;
+        }
+    } else {
+        colorBrush = rt_dynamic_cast<WUXMSolidColorBrush>(_textBox.background);
+        if (colorBrush != nil) {
+            return ConvertWUColorToUIColor(colorBrush.color);
+        } else {
+            return nil;
+        }
+    }
 }
 
 /**
  @Status Interoperable
 */
 - (void)setTextAlignment:(UITextAlignment)alignment {
-    _alignment = alignment;
-
     [_secureModeLock lock];
     if (!_secureTextMode) {
         // passwordBox does not support text alignment
-        _textBox.textAlignment = ConvertUITextAlignmentToWXTextAlignment(_alignment);
+        _textBox.textAlignment = ConvertUITextAlignmentToWXTextAlignment(alignment);
     }
```

Comment:
good idea

---
In reply to: [95474383](https://github.com/Microsoft/WinObjC/pull/1604#discussion_r95474383) [](ancestors = 95474383)